teamA/kimTaeksoo/service/postService.js

```diff
@@ -0,0 +1,18 @@
+const postDao = require("../model/postDao");
+
+const posting = async (title, content, userId, postingImageUrl) => {
+  const createPost = await postDao.createPost(
+    title,
+    content,
+    userId,
+    postingImageUrl
+  );
+  return createPost;
+};
+
+const allPosts = async () => {
+  const data = await postDao.allPosts();
+  return data;
```

Comment:
Dao에서 이미 데이터를 반환하고 있기 때문에 한 줄로 리팩토링 할 수 있을 것 같습니다.
```suggestion
  return await postDao.allPosts();
```